src/components/families/FamilyDetailsSidebar.tsx

```diff
@@ -80,6 +86,18 @@ const FamilyDetailsSidebar = ({
               {family.parent.information[parentField.id]}
             </Typography>
           ))}
+          <Typography variant="body2" className={classes.heading}>
+            Notes
+          </Typography>
+          <form>
+            <TextField
```

Comment:
textfields need a unique id on it for accessibility ([docs](https://material-ui.com/components/text-fields/#accessibility))! could we add `id={DefaultFieldKey.NOTES}`?